gluten-ut/spark33/src/test/scala/org/apache/spark/sql/gluten/GlutenFallbackSuite.scala

```diff
@@ -93,6 +93,18 @@ class GlutenFallbackSuite extends GlutenSQLTestsTrait with AdaptiveSparkPlanHelp
       }
     }
 
+    withTable("t1", "t2") {
+      spark.range(10).write.format("parquet").saveAsTable("t1")
+      spark.range(10).write.format("parquet").saveAsTable("t2")
+
+      val id = runExecution("SELECT * FROM t1 FULL OUTER JOIN t2")
+      val execution = glutenStore.execution(id)
+      assert(execution.get.numFallbackNodes == 1)
+      assert(
+        execution.get.fallbackNodeToReason.head._2
+          .contains("FullOuter join is not supported with Broadcast Nested Loop Join"))
```

Comment:
Nit: Probably we should use consistent casing between "full-outer join" and "broadcast nested-loop join"?